ingest-app/src/cmr/ingest/api/bulk.clj

```diff
@@ -34,6 +34,22 @@
           {:status 200
            :task-id task-id})))))
 
+(defn bulk-update-granules
+  "Bulk update granules. Validatie prvider exists, check ACLs, and validate
+   POST body. Writes tasks to queue for worker to begin processing."
+  [provider-id request]
+  (if-not (ingest-config/granule-bulk-update-enabled)
```

Comment:
Should update `granule-bulk-update-enabled` name to end with `?`

Reply:
The other config value didn't end with `?`, but I like that better